typescript/sdk/src/providers/MultiProvider.ts

```diff
@@ -230,7 +230,7 @@ export class MultiProvider {
 
     if (TestChains.includes(name as CoreChainName)) {
       this.providers[name] = new providers.JsonRpcProvider(
-        'http://localhost:8545',
+        'http://127.0.0.1:8545',
```

Comment:
not sure why you needed to change this everywhere

Reply:
wasn't working with anvil or hardhat, with error `ECONNREFUSED` from the rpc.